packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -31,6 +38,12 @@ When('I type a new {string}', (loginMechanism: string) => {
   );
 });
 
+When('I click the {string} tab', (label: string) => {
```

Comment:
@slaymance I made this more abstract to support I18n instead of `I click the Create Account tab`.